metis-dereference/metis-dereference-common/src/main/java/eu/europeana/metis/dereference/Vocabulary.java

```diff
@@ -62,15 +83,26 @@
   @Indexed(options = @IndexOptions(unique = true))
   private String name;
 
-  private ContextualClass type;
-
   @XmlElement
-  public String getUri() {
-    return uri;
+  public Set<String> getUris() {
+    return Collections.unmodifiableSet(this.uris);
   }
 
-  public void setUri(String uri) {
-    this.uri = uri;
+  public void setUris(Collection<String> uris) {
+    this.uris = new HashSet<>(uris);
+
+    // Also take care of backwards compatibility
```

Comment:
How about we encapsulate this in a private method and mark it as deprecated so that we know we need to remove it?

Reply:
Sure.